projects/owa-env-desktop/owa/env/desktop/msg.py

```diff
@@ -48,6 +48,7 @@ class WindowInfo(OWAMessage):
     # 0 <= top < bottom <= screen_height
     rect: tuple[int, int, int, int]
     hWnd: int
+    window: Any | None = None
```

Comment:
Since `WindowInfo` object is recorded also in OWMcap file, it's definition must be clear and strict. Rolling back this line will be good,

Reply:
Reason: for make_active()
DONE: Remove `window` for WindowInfo, directly use pygetwindow in make_active()